Assets/MixedRealityToolkit/InputSystem/Cursors/BaseCursor.cs

```diff
@@ -134,6 +134,7 @@ public bool IsVisible
         public virtual void OnSourceDetected(SourceStateEventData eventData)
         {
 #if UNITY_WSA
+            // TODO Figure out a better way to know what type of source is found / lost.
             InteractionSourceKind sourceKind;
```

Comment:
Note, Multi-VR breaking issue

Reply:
Not exactly a breaking issue, just doesn't currently conform to our standards. Let's add an issue and get back to this later. It will compile.

Reply:
Ok, agreed. Create a task for an #IF UNITY pass to remove these.

but if I switched to standalone, this wouldn't work.

Reply:
Well you wouldn't be using hand input on standalone without a Magic Leap right? 😉